src/helpers/syntax-highlighting.js

```diff
@@ -51,7 +51,7 @@ hljs.registerLanguage('xml', xml);
 
 export default function syntaxHighlighting(string, lang) {
   if (!lang || !hljs.getLanguage(lang)) {
-    return '';
+    return string;
```

Comment:
@johncorrelli had to change this so that it wouldn't break when there wasn't a language set.